sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -176,170 +176,237 @@ public void BuildModelError()
         [RecordedTest]
         [TestCase(true)]
         [TestCase(false)]
-        public async Task CopyModelTo(bool useTokenCredential)
+        public async Task CopyDocumentModelTo(bool useTokenCredential)
         {
-            var sourceClient = CreateDocumentModelAdministrationClient(useTokenCredential);
-            var targetClient = CreateDocumentModelAdministrationClient(useTokenCredential);
+            var client = CreateDocumentModelAdministrationClient(useTokenCredential);
+            var sourceModelId = Recording.GenerateId();
             var modelId = Recording.GenerateId();
+            var description = $"This model was generated by a .NET test: {nameof(CopyDocumentModelTo)}";
+            var tags = new Dictionary<string, string>() { { "tag1", "value1" }, { "tag2", "value2" } };
+            var startTime = Recording.UtcNow;
 
-            await using var trainedModel = await BuildDisposableDocumentModelAsync(modelId);
+            await using var disposableModel = await BuildDisposableDocumentModelAsync(sourceModelId);
 
-            var targetModelId = Recording.GenerateId();
-            DocumentModelCopyAuthorization targetAuth = await targetClient.GetCopyAuthorizationAsync(targetModelId);
+            DocumentModelCopyAuthorization copyAuthorization = await client.GetCopyAuthorizationAsync(modelId, description, tags);
+            CopyDocumentModelToOperation operation = null;
 
-            CopyDocumentModelToOperation operation = await sourceClient.CopyDocumentModelToAsync(WaitUntil.Completed, trainedModel.ModelId, targetAuth);
+            try
+            {
+                operation = await client.CopyDocumentModelToAsync(WaitUntil.Completed, sourceModelId, copyAuthorization);
+            }
+            finally
+            {
+                if (operation != null && operation.HasValue)
+                {
+                    await client.DeleteDocumentModelAsync(modelId);
+                }
+            }
```

Comment:
We had a model leak in this test. The flow of the test was basically:
1. Create model A.
2. Copy model A into model B.
3. Do some validation.
4. Delete model A.
We were forgetting to delete model B, copied from A.